### PR TITLE
fix: Login password length upper limit changed from 48 to 256 characters

### PR DESCRIPTION
**Pull request by ankitsrivas14**
## Description

Changed the login password length upper limit from 48 to 256 characters

Fixes # (issue)
[[Bug]-[672]:Passwords longer than 48 characters are "not strong enough](https://github.com/appsmithorg/appsmith/issues/12965)

### Before
image

### After
image

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/utils/formhelpers.ts [hunk 1]
```diff
@@ -11,10 +11,7 @@ export const isEmptyString = (value: string) => {
 
 export const isStrongPassword = (value: string) => {
   const passwordLength = value.trim().length;
-  return (
-    passwordLength >= PASSWORD_MIN_LENGTH &&
-    passwordLength < PASSWORD_MAX_LENGTH
-  );
+  return passwordLength >= PASSWORD_MIN_LENGTH;
```
Comment (IAmAnubhavSaini) [inline app/client/src/utils/formhelpers.ts:14]: Might need to delete `const PASSWORD_MAX_LENGTH = 48;` from above in the file. Check if that value is used anywhere in all of our repos.

Comment (IAmAnubhavSaini) [reply app/client/src/utils/formhelpers.ts:14]: We can further simplify the function. `export const isStrongPassword = (value: string) => value.trim().length >= PASSWORD_MIN_LENGTH`.

**Review by IAmAnubhavSaini — CHANGES_REQUESTED** [review]

File: app/client/src/utils/formhelpers.ts (hunk 1 above)
Comment (ankitsrivas14) [reply app/client/src/utils/formhelpers.ts]: @IAmAnubhavSaini  -- Done.

**Review by IAmAnubhavSaini — APPROVED** [review]

**Review by IAmAnubhavSaini — CHANGES_REQUESTED** [review]
@Ankit Srivastava Please make max limit 256 characters long.

**Comment by ankitsrivas14** [comment]
> @ankit Srivastava Please make max limit 256 characters long.

Done

**Review by IAmAnubhavSaini — APPROVED** [review]

**Comment by IAmAnubhavSaini** [comment]
Update the branch with release changes and trigger testing.

**Pull request merged**